packages/wonder-blocks-button/components/button-core.js

```diff
@@ -104,48 +86,29 @@ export default class ButtonCore extends React.Component<Props> {
                         style={sharedStyles.icon}
                     />
                 )}
-                {children}
+                <View style={sharedStyles.parentWidth}>{children}</View>
             </Label>
         );
-
-        const contents = (
-            <React.Fragment>
+        return (
+            <View style={[defaultStyle]}>
```

Comment:
Same here with using `View` (div).

Reply:
I would need a way to apply the style as above